Index: String_Methods/Join_Method.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- String_Methods/Join_Method.py	(revision )
+++ String_Methods/Join_Method.py	(revision )
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# -*- coding:utf-8 -*-
+"""
+string.join(self, iterable)
+    Return a string which is the concatenation of the strings in iterable. A TypeError will be raised if there are any non-string values in iterable, including bytes objects. The separator between elements is the string providing this method.
+    以 string 作为分隔符，将 seq 中所有的元素(的字符串表示)合并为一个新的字符串.
+"""
+__author__ = "Curry"
+
+# example
+seq = [1, 2, 3, 4, 5]
+sep = '+'
+print(sep.join(seq))
+# TypeError: sequence item 0: expected str instance, int found
+
+# fix
+seq = ['1', '2', '3', '4', '5']
+sep = '+'
+print(sep.join(seq))
+# 1+2+3+4+5
+
+# example
+dirs = '', 'usr', 'bin', 'env'
+print('/'.join(dirs))
+# /usr/bin/env
+
+# example
+print('C:' + '\\'.join(dirs))
+# C:\usr\bin\env
